### DISCUSSION TITLE
Unable to access email of user with a prisma database adapter enabled

### DISCUSSION THREAD

**Discussion by SamKingDev** [nextauthjs/next-auth#4939] [Help]
Hello,

If I enable a Prisma database adapter then the email field of the useSession() data object becomes null every time?

I feel like I'm missing some fairly obvious knowledge here.

Also, should note that the email isn't being saved to the database either.

How can I go about ensuring the email is captured?

Here is my [...nextauth].ts

```ts
import { PrismaAdapter } from "@next-auth/prisma-adapter";
import { PrismaClient } from "@prisma/client";
import NextAuth from "next-auth";
import GithubProvider from "next-auth/providers/github";

const prisma = new PrismaClient();

export default NextAuth({
  adapter: PrismaAdapter(prisma),
  providers: [
    GithubProvider({
      clientId: process.env.GITHUB_ID,
      clientSecret: process.env.GITHUB_SECRET,
      authorization: `https://github.com/login/oauth/authorize?client_id=${process.env.GITHUB_ID}&redirect_uri=${process.env.NEXTAUTH_URL}/api/auth/callback/github&allow_signup=true&scope=user:email`,
    }),
  ],
});
```

**Comment by ThangHuuVu** [comment]
Make sure you are on `next-auth@latest`, we've fixed [this issue](https://github.com/nextauthjs/next-auth/issues/4934) recently 👀
